### ISSUE TITLE
Packaging?

### ISSUE THREAD

**Issue by jbergstroem** [attractivechaos/klib#1]
Any plans on doing releases of klib? Versioning, tarballing and so on. I'd really like to see changelog notes outside the header files as well.

**Comment by theduke** [comment]
I wouldn't care about packaging, but versioning is important.

I woud recommend semantic versioning (http://semver.org/).

**Comment by martingkelly** [comment]
+1 I for one would love packaging. I think a simple package that just installs these the headers into a klib subdir would be super handy.

**Comment by solvingj** [comment]
There are reasonable arguments for consuming this library as single files on-demand, as well as for consuming it as a package. My team would like to create a package for this library for the Conan package manager, however things are much less attractive if we don't have a semantically versioned github release/tarball that we can pin to, and increment over time as the components evolve.

**Comment by martingkelly** [comment]
Releases would be nice, I agree. If there are none, you can also use an auto-incrementing number + a git commit hash as the version. Debian does this for some packages.

**Comment by solvingj** [comment]
Yes, we have a documented strategy for it here, but as I said, it’s suboptimal for us and consumers.

https://bincrafters.readthedocs.io/en/latest/using_packages.html#packages-without-official-releases
